src/tr1/game/objects/creatures/statue.c

```diff
@@ -49,7 +49,7 @@ void Statue_Initialise(int16_t item_num)
     Item_Initialise(centaur_item_num);
 
     Item_SwitchToAnim(centaur, CENTAUR_REARING_ANIM, CENTAUR_REARING_FRAME);
-    centaur->current_anim_state = g_Anims[centaur->anim_num].current_anim_state;
+    centaur->current_anim_state = Item_GetAnim(centaur)->current_anim_state;
```

Comment:
Am I crazy or is this doing nothing? It's setting `centaur->current_anim_state` to `centaur`'s `current_anim_state`?

Reply:
It's setting the item's state to that of the animation, which is 6 in this case so it's in the rearing state.